src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
Perhaps header comment could be added to describe what this interface represents?
Recall from tutorial that an interface is a behavioural contract.
Perhaps `Represents a printable entity.....`?